### PR TITLE
OSDOCS 11357: Updated ImageSet Config example to include KubeVirt CoreOS container images in oc-mirror

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-11357

Link to docs preview: https://81280--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/disconnected_install/about-installing-oc-mirror-v2#oc-mirror-imageset-config-parameters-v2_about-installing-oc-mirror-v2

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oc-mirror-imageset-config-parameters-v2.adoc [hunk 1]
```diff
@@ -35,6 +35,23 @@ Example: `mirror.openshift.io/v2alpha1`
 |Integer 
 Example: `4`
 
+|`kubeVirtContainer`
+|When `true`, includes images from the HyperShift KubeVirt CoreOS container.
```
Comment (kasturinarra) [inline modules/oc-mirror-imageset-config-parameters-v2.adoc:39]: @subhtk  I think we should say like `when true includes images from release payload for HyperShift KubeVirt CoreOS container` ?

@sherine-k WDYT ?

Comment (sherine-k) [reply modules/oc-mirror-imageset-config-parameters-v2.adoc:39]: sounds good!

**Review by opayne1 — COMMENTED** [review]
Don't forget to update the preview link but otherwise lgtm 

https://81280--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/disconnected_install/about-installing-oc-mirror-v2.html#:~:text=Table%202.%20ImageSetConfiguration%20parameters

File: modules/oc-mirror-imageset-config-parameters-v2.adoc [hunk 2]
```diff
@@ -35,6 +35,23 @@ Example: `mirror.openshift.io/v2alpha1`
 |Integer 
 Example: `4`
 
+|`kubeVirtContainer`
+|When `true` includes images from the HyperShift KubeVirt CoreOS container.
+|Boolean
+Example `ImageSetConfiguration` file
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/oc-mirror-imageset-config-parameters-v2.adoc:41]: Suggestion to add a colon `:` to be consistent with the other examples in the table

**Review by xenolinux — COMMENTED** [review]
Two small nits before merging! Otherwise looks good.

To merge the PR, feel free to ping me or set the label again!

/remove-label merge-review-needed
/remove-label merge-review-in-progress

File: modules/oc-mirror-imageset-config-parameters-v2.adoc [hunk 3]
```diff
@@ -35,6 +35,23 @@ Example: `mirror.openshift.io/v2alpha1`
 |Integer 
 Example: `4`
 
+|`kubeVirtContainer`
+|When `true` includes images from the HyperShift KubeVirt CoreOS container.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/oc-mirror-imageset-config-parameters-v2.adoc:39]: Ack.

File: modules/oc-mirror-imageset-config-parameters-v2.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/oc-mirror-imageset-config-parameters-v2.adoc:41]: Ack.

**Comment by xenolinux** [comment]
/label merge-review-in-progress

Waiting for the checks to pass

**Pull request merged**